/* 2016-03-16 */
theApp.controller("loginCtrl",function($scope,$auth){$scope.login=function(){$auth.submitLogin($scope.loginForm).then(function(resp){console.log("Logged in!?")})["catch"](function(resp){console.log("Obviously got an error response!"),$scope.loginErrorMessage="Sorry, but uh we couldn't find you, try again"})}});